### PR TITLE
Add missing dialog frame for Hero Meeting dialog

### PR DESCRIPTION
**Pull request by ihhub**
Somehow it has been always absent.

image

### CODE REVIEW

**Comment by oleg-derevenetz** [comment]
Now it looks like there are two frames.

**Review by zenseii — COMMENTED** [review]
It works just as the engine does already for the Battle Only screen, but as @oleg-derevenetz noted it does look like two frames.

Not sure how much work it is and if there's reason to put effort into it here, but the inner frame is a "good" type frame as seen by the interface type selection icon:
image
The evil frame would be the one that looks like this I imagine:
image

**Comment by oleg-derevenetz** [comment]
@ihhub

> if you click on one of heroes without these changes you would get an idea :)

Ah, yes, that's right.

File: src/fheroes2/heroes/heroes_meeting.cpp [hunk 1]
```diff
@@ -225,13 +226,18 @@ class MeetingSecondarySkillsBar : public SecondarySkillsBar
...
 {
     fheroes2::Display & display = fheroes2::Display::instance();
 
     // setup cursor
     const CursorRestorer cursorRestorer( true, Cursor::POINTER );
 
+    std::unique_ptr<fheroes2::StandardWindow> background;
+    if ( renderBackgroundDialog ) {
+        background = std::make_unique<fheroes2::StandardWindow>( fheroes2::Display::DEFAULT_WIDTH, fheroes2::Display::DEFAULT_HEIGHT );
+    }
```
[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/heroes/heroes_meeting.cpp:239]: We will draw the dialog's shadow twice. #5590 fixed this part.

**Comment by ihhub** [comment]
I am marking this pull request for 0.9.21 release as we need to fix double-frame issue.

**Comment by ihhub** [comment]
Closing this PR due to invalid solution.

**Pull request closed without merging**